user_manual/files/access_webdav.rst

```diff
@@ -439,6 +439,28 @@ Solution
 
 Workarounds are documented in the KB2668751_ article.
 
+Problem
+^^^^^^^
+Error 0x80070043 "The network name cannot be found." while adding a network drive.
+
+Solution
+^^^^^^^^
+Make Windows service **WebClient** starts "Automatically":
```

Comment:
s/starts/start/
and I guess "Automatically" could be just automatically (without quotes or capitalization)